docs/metachromium-overview.md

```diff
@@ -3,5 +3,54 @@ id: metachromium-overview
 title: Metachromium Overview
 ---
 
-Metachromium
+[Metachromium](https://metachromium.com/) is a spatial app browser that runs side-by side with OpenVR. It lets you open any WebXR page as a spatial overlay app that runs on top of any other VR game.
 
+## Features
+
+- Run WebXR context on top of an OpenVR application
+
+- Run multiple WebXR applications inside a single Metachromium context
+  
+- Add multiplayer to singleplayer apps using metaRTC
+
+- Volumetric streaming capabilities
+
+## Using Metachromium
+
+### Prerequisites
+
+- XR Device (for best experience)
+
+- At least 1GB of storage
+
+### Running
+
+1. Install Metachromium through [Steam](https://store.steampowered.com/app/685110/Metachromium/). If you need an early-access key, get one [here](https://metachromium.com/).
+
+2. Run your SteamVR application.
+
+3. Run Metachromium and choose an app you would like to overlay.
+
+4. Enjoy!
+
+## How It Works
+
+Metachromium is based on the open source Chromium browser and is based on web standards like WebXR.
+
+### Rendering Content
+
+For different WebXR contexts to be overlayed inside an OpenVR application, special math must be done so the outlines of these different applications blend into the world correctly in terms of its alpha (opacity). Metachromium does the blending such that the renderings appear as though they are in the correct space when viewed through the headset. Functionally, there are a bunch of 3D geometry calculations that are done under the hood to make an immersive experience possible.
```

Comment:
I believe we can be more specific if we're going to give a technical explanation. Metachromium is capturing a 360 stereoscopic view from the WebXR frame buffer and then mapping that view to a transparent sphere that surrounds the user. A bit like a dynamic photosphere. [Here is a link](https://youtu.be/YGsXRSEemRw?t=1896) to the timestamp where Avaer covered this in the overview